sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormField.cs

```diff
@@ -163,5 +162,29 @@ private static FormElement ResolveTextReference(IReadOnlyList<ReadResult> readRe
 
             throw new InvalidOperationException($"Failed to parse element reference: {reference}");
         }
+
+        /// <summary>
+        /// Business Cards pre-built model doesn't return a page number for the `ContactNames` field.
+        /// This function looks into the FieldValue_internal to see if it corresponds to
+        /// `ContactNames` and verifies that the page value before returning it.
+        /// </summary>
+        /// <returns>Page value if the field is `ContactNames` for Business cards. If not, defaults to 1.</returns>
+        private static int CalculatePage(FieldValue_internal field)
```

Comment:
what if instead of running this check for every FormField, we pass down a flag from the business card method that indicates that this is a business card and we do need to calculate the page for ContactNames?

Reply:
Good idea. I will try that

Reply:
Updated

Reply:
>this behavior only exists on "ContactNames" field?

Yes.

>Is this a service bug that they are planning to update later?

It is not a service bug. We are asking them to consider adding the `Page` field so we don't have to do all of this. But they haven't agreed. Conversation is here => https://teams.microsoft.com/l/message/19:88f2d9dea2344075b5d4bc34d82a3d1c@thread.skype/1602281344886?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1602281344886&teamName=Azure%20SDK&channelName=Service%20-%20Form%20Recognizer&createdTime=1602281344886

Reply:
I updated the description of the PR to include this change too.